src/components/Game/Letters/Letters.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classes from './Letters.scss';
+
+const Letters = () => {
+    const letterClasses = [
+        classes.Letter,
+        classes.Inactive,
+    ];
+
+    return (
+        <section className={classes.Letters}>
+            <div className={classes.LettersRow}>
+                <div className={letterClasses.join(' ')}>a</div>
+                <div className={classes.Letter}>s</div>
+            </div>
+            <div className={classes.LettersRow}>
+                <div className={letterClasses.join(' ')}>t</div>
+                <div className={classes.Letter}>r</div>
+                <div className={classes.Letter}>k</div>
+            </div>
+            <div className={classes.LettersRow}>
+                <div className={letterClasses.join(' ')}>l</div>
```

Comment:
regarding these low-level `.join(' ')`, one may say it's a little bit imperative/ugly
there is a npm package that may help a bit https://www.npmjs.com/package/classnames in this case and in more complex circumstances as well
please have a look and decide if you like it